packages/orchestration/src/exos/local-chain-account-kit.js

```diff
@@ -54,16 +83,21 @@ export const prepareLocalchainAccountKit = (baggage, makeRecorderKit, zcf) => {
       }),
     },
     /**
-     * @param {import('@agoric/vats/src/localchain.js').LocalChainAccount} account
-     * @param {StorageNode} storageNode
+     * @param {object} initState
+     * @param {LocalChainAccount} initState.account
+     * @param {ChainAddress['address']} initState.address
+     * @param {StorageNode} initState.storageNode
+     * @param {TimerService} initState.timer
+     * @param {TimerBrand} initState.timerBrand
+     * @param {AgoricChainInfo} initState.agoricChainInfo
```

Comment:
these aren't specific to each account kit, right? they can stay in the `prepareLocalChainAccountKit` closure.

Reply:
Good call out, thank you